src/_cffi_src/openssl/ec.py

```diff
@@ -182,25 +114,8 @@
 #if defined(OPENSSL_NO_EC2M)
 static const long Cryptography_HAS_EC2M = 0;
 
-const EC_METHOD *(*EC_GF2m_simple_method)() = NULL;
-
-int (*EC_POINT_set_affine_coordinates_GF2m)(const EC_GROUP *, EC_POINT *,
```

Comment:
This was removed here, but still exists in another place in this file, leading to an error loading the .so